src/niscope/examples/niscope_fetch_grpc.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python
+
+import argparse
+import grpc
+import niscope
+import pprint
+import sys
+
+pp = pprint.PrettyPrinter(indent=4, width=80)
+
+
+def example(resource_name, channels, options, length, voltage, address, port):
+    session_name = '' # user-specified name; empty string means use a new, unnamed session
```

Comment:
Sorry, my initial comment was a bit off with some of the spacing. I updated it, but it looks like you committed the original. You may need to readd that second space before the comment to pass the flake8 check. If not, it's fine as is.

Reply:
Yeah, you'll need to readd that second space.